.huskyrc

```diff
@@ -0,0 +1 @@
+HUSKY_USE_YARN=1
```

Comment:
This doesn't seem to do anything anymore https://github.com/typicode/husky/search?q=HUSKY_USE_YARN&type=Code